ReactiveCocoa/Swift/Property.swift

```diff
@@ -366,17 +366,15 @@ public struct AnyProperty<Value>: PropertyType {
 	/// Initializes a property that first takes on `initialValue`, then each value
 	/// sent on a signal created by `producer`.
 	public init(initialValue: Value, producer: SignalProducer<Value, NoError>) {
-		let mutableProperty = MutableProperty(initialValue)
-		mutableProperty <~ producer
-		self.init(mutableProperty)
+		self.init(propertyProducer: SignalProducer(value: initialValue).concat(producer),
```

Comment:
I just updated the `RAC5` branch from `master`. Can you merge into this branch and use the new `prefix` operator here? It will simplify it quite a bit.